lesson3/ToDoList2/app/src/main/java/erica/todolist2/MainActivity.java

```diff
@@ -16,20 +16,18 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);                                                     //gets the layout for the view from activity main
-        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);                                     //defines the toolbar
-        setSupportActionBar(toolbar);                                                               //makes the toolbar
+        setContentView(R.layout.activity_main);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
 
         final Fragment listFrag = new ListFragment();
-        replaceFragment(listFrag);                                                                  //calls the replaceFragment method defined below for the current view to be the list fragment
+        replaceFragment(listFrag); // default view is the list fragment
```

Comment:
Also, when you're doing in-line comments, don't put so much space in between the end of your code line and the start of your comment.  See here, I just have one space and then start my comment.